Answers.md

```diff
@@ -8,4 +8,47 @@ Scott Grobe project
 
 4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
 
-5. What is the git command to commit staged changes as well as write a message? 
\ No newline at end of file
+5. What is the git command to commit staged changes as well as write a message? 
+
+
+
+
+
+
+
+questions on git-hub are different..
+
+1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+
+answer:  Semantic HTML provides MEANING to the content so its easier for machines and for humans to understand.
+
+
+
+
+
+2. Name two big differences between display: block; and display: inline;.
+
+answer: 
+
+        INLINE: top and bottom margins, height and width settings are not respected, Inline is the defafult, Inline is displayed left-to right
```

Comment:
block takes up full width of parent element, inline allows for other elements to be placed next to each other